com.microsoft.mrtk.graphicstools.unity/Runtime/Utilities/DrawFullscreenFeature.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#if GT_USE_URP
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace Microsoft.MixedReality.GraphicsTools
+{
+    public enum BufferType
+    {
+        CameraColor,
+        Custom 
+    }
+
+    /// <summary>
+    /// Forked from: https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/master/Assets/Scripts/Runtime/RenderPasses
+    /// </summary>
+    public class DrawFullscreenFeature : ScriptableRendererFeature
+    {
+        [System.Serializable]
+        public class Settings
```

Comment:
nit: can you add [API reference](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-graphicstools/contributing#api-reference-documentation) for all public properties and methods?